src/check-npm-install-util.js

```diff
@@ -21,7 +21,10 @@ try {
     // Ensure npm install has been run.
     Object.keys(require('../package').dependencies).forEach(require);
 } catch (e) {
-    console.log('Please run "npm install" from this directory:\n\t' + __dirname); // eslint-disable-line no-path-concat
+    const path = require('path'); // Built-in Node.js module
+    console.log(
+       'Please run "npm install" from this directory:\n\t' +
+        path.join(__dirname, '..')); // correct path
```

Comment:
Why the added `..` here? 
Can't coho be installed/run globally as well? Or does this not matter then because `npm install` should have been run?
What about running the coho js script manually from different locations?

Reply:
> Why the added `..` here?

For some reason if I don't add the `..` then it would show "/src" at the end, on my mac at least.

> Can't coho be installed/run globally as well? Or does this not matter then because `npm install` should have been run?

Yes to the 2nd question. Or let me say that it *should* not matter since `npm install` *should* have been run _correctly_.

> What about running the coho js script manually from different locations?

I tried it from several different places, seems to be correct with the proposed fix and show extra "/src" at the end without this proposed fix.